source/_docs/configuration/templating.markdown

```diff
@@ -483,6 +484,17 @@ Closest to some entity:
 
 Some of these functions can also be used in a [filter](https://jinja.palletsprojects.com/en/latest/templates/#id11). This means they can act as a normal function like this `sqrt(2)`, or as part of a filter like this `2|sqrt`.
 
+<div class='note'>
+
+The numeric functions and filters will not fail if the input is not a valid number, instead the input value will be returned with the exception of the flaot fitler which returns `0.0`. This is unwanted behavior in many cases, use `is_number` to check if the value is valid.
+
+`{{ float("not_a_number") }}` - renders as `"not_a_number"`
+`{{ "not_a_number" | float }}` - renders as `0.0`
+`{{ sin("not_a_number") }}` - renders as `"not_a_number"`
+`{{ "not_a_number" | sin }}` - renders as `"not_a_number"`
```

Comment:
Should this be expanded with some examples of checking the value first?

Reply:
I think this is fine. In general, most template examples can/could be extended to catch a lot of edge cases, but that would make the docs harder to read.